pages.de/linux/ip.md

```diff
@@ -25,7 +25,7 @@
 
 - Schalte ein bestimmtes Interface ein oder aus:
 
-`ip link set {{interface}} {{up|down}}`
+`ip link set {{interface}} up/down`
```

Comment:
this needs to remain a placeholder since the user has the choice between uo and down and cannot specify the literal string `up/down`